src/appengine/handlers/cron/triage.py

```diff
@@ -250,11 +257,105 @@ def _check_and_update_similar_bug(testcase, issue_tracker):
   return False
 
 
-def _file_issue(testcase, issue_tracker):
+def _query_job_bugs_filed_count(job_type, valid_timestamp):
+  """Gets the number of bugs that have been filed for a given job
+  within a given time period."""
+  return data_types.FiledBug.query(
+      data_types.FiledBug.job_type == job_type and
+      data_types.FiledBug.timestamp >= valid_timestamp).count()
+
+
+def _query_project_bugs_filed_count(project_name, valid_timestamp):
+  """Gets the number of bugs that have been filed for a given project
+  within a given time period."""
+  return data_types.FiledBug.query(
+      data_types.FiledBug.project_name == project_name and
+      data_types.FiledBug.timestamp >= valid_timestamp).count()
+
+
+def _get_job_bugs_filing_max(job_type):
+  """Gets the maximum number of bugs that can be filed for a given job."""
+  if job_type in _bug_filing_max_24_hours_per_job:
+    return _bug_filing_max_24_hours_per_job[job_type]
+
+  max_bugs = None
+  job = data_types.Job.query(data_types.Job.name == job_type).get()
+  if job and 'BUG_FILING_MAX_24_HOURS_PER_JOB' in job.get_environment():
+    try:
+      max_bugs = int(job.get_environment()['BUG_FILING_MAX_24_HOURS_PER_JOB'])
+    except Exception:
+      logs.log_error(
+          'Invalid environment value of \'BUG_FILING_MAX_24_HOURS_PER_JOB\' '
+          f'for job type {job_type}.')
+
+  _bug_filing_max_24_hours_per_job[job_type] = max_bugs
+  return max_bugs
+
+
+@lru_cache(1)
+def _get_project_bugs_filing_max():
+  """Gets the maximum number of bugs that can be filed per project."""
+  config = local_config.IssueTrackerConfig()
+  try:
+    return int(
+        config.get('BUG_FILING_MAX_24_HOURS_PER_PROJECT',
```

Comment:
You need to get the issue tracker project first. see e.g. https://github.com/google/clusterfuzz/blob/992274ebd11e178b2c664ec2fa684ff7d98df376/src/appengine/libs/issue_management/issue_tracker_policy.py#L181.

You can get this from https://github.com/google/clusterfuzz/blob/992274ebd11e178b2c664ec2fa684ff7d98df376/src/clusterfuzz/_internal/datastore/data_handler.py#L682. 

Also let's not make this variable ALL_CAPS, since it doesn't match the existing convention.